django/utils/html.py

```diff
@@ -77,6 +78,28 @@ def escapejs(value):
     return mark_safe(str(value).translate(_js_escapes))
 
 
+_json_tag_escapes = {
+    ord('>'): '\\u003E',
+    ord('<'): '\\u003C',
+    ord('&'): '\\u0026',
+}
+
+
+def json_tag(value, element_id):
+    """
+    This will escape all the HTML/XML special characters with their unicode
+    escapes, so it is safe to be output anywhere except for inside a tag
+    attribute. It will wrap the escaped json in a script tag.
+    See https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet, rule #3.1 for details.
```

Comment:
I'd drop that comment since it doesn't really escape according to the rules suggested there.